dsl-reference.md

```diff
@@ -176,7 +176,7 @@ do:
       - getAvailablePets:
```

Comment:
Idk why, I cannot comment the beginning of the time, but `export` is missing in the TOC.